tensorrt_llm/_torch/modules/kimi_k3_attn_res/_attn_res_kernels.py

```
@@ -0,0 +1,82 @@
+Dispatch: on sm_100/sm_103 with the compiled Torch bindings loaded the
+module uses the fused op; otherwise it falls back to the pure-torch chunked
+reference in :mod:`kimi_k3_attn_res.kimi_k3_attn_res`.
+"""
+
+from __future__ import annotations
+
+import torch
+
+try:
+    from tensorrt_llm._utils import get_sm_version as _tllm_get_sm_version
+except Exception:  # pragma: no cover — source-loader stub path
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)